moduleroot/CONTRIBUTING.md.erb

```diff
@@ -135,9 +135,19 @@ Our puppet modules provide [`Gemfile`](./Gemfile)s which can tell a ruby
 package manager such as [bundler](https://bundler.io/) what Ruby packages,
 or Gems, are required to build, develop, and test this software.
 
-Please make sure you have [bundler installed](https://bundler.io/#getting-started)
-on your system, then use it to install all dependencies needed for this project,
-by running
+**Prerequisites**
+1. Make sure you have [bundler installed](https://bundler.io/#getting-started)
+on your system. If you are using Fedora, you can get `bundler` using
+   ```shell
+     % sudo dnf install rubygem-bundler
```

Comment:
I know `#` is root and `$` is a normal user but what's the `%` convention? I do see it's used throughout the file so I guess I never really read this.

Reply:
My zsh also uses `%` for user ;)

Reply:
yeah, I am new to `%` as well, but thank you @evgeni for the info.

@ekohl I think we mostly use `$` and `#` in all other documentation, I am happy to fix it in this doc in a separate issue.

Reply:
@ekohl Actually I saw forklift docs, we don't use either if not required. So I removed `%` from the places where it is not really required.  I replaced `%' with `$` in just one instance where it makes sense to show it.